### DISCUSSION TITLE
line-chart: how to have the min value at the top?

### DISCUSSION THREAD

**Discussion by stephanegrenier** [nhn/tui.chart#680] [Q&A]
Hi,

I want to use the line-chart but in the reverse order.
I want to have the min value at the top of the view and the max value at the bottom
I tried to something like this but it doesn't work
`yAxis: {
          title: 'Ranking',
		  scale: {
			min: 20,
			max: 1,
			stepSize: -1
		  }
        },`

I want to have 1 at the top and 20 at the bottom to display a ranking line

Thanks

**Comment by jung-han** [comment]
Oh, that part is not developed at all. It is natural to look wrong. 
Could you please provide some data and a simple form on what form the chart should appear in?

**Reply by stephanegrenier** [reply]
So here is my data:
[data.txt](https://github.com/nhn/tui.chart/files/6662520/data.txt)

I have something like this:
<img width="1214" alt="image" src="https://user-images.githubusercontent.com/83098639/122209117-70f51380-cea4-11eb-896c-fc1b27cda7ca.png">

but I would like something like this:
![image](https://user-images.githubusercontent.com/83098639/122211319-e366f300-cea6-11eb-9c01-0fa1982a48c9.png)

So the best team at the top of the graph :)

**Reply by stephanegrenier** [reply]
Hi @jung-han,
any updates?
Do you need more information?

**Reply by jwlee1108** [reply]
@stephanegrenier 

Hello, I'm sorry, but can I proceed with this problem a little later? 🙏
There are a lot of issues that I'm dealing with now.

But, I will check this matter again as soon as possible.

**Reply by stephanegrenier** [reply]
@jwlee1108 
Of course,I understand.

This is not a bug but an enhancement, this is less priority.
Keep me in touch when it is implemented.

Thanks

**Reply by stephanegrenier** [reply]
Hello,
I come to the news about my request
Thanks

**Reply by stephanegrenier** [reply]
Hello @jwlee1108 
Any update?
